Scaffolding a Grunt project fails on OSX 10.11.1 and Node 5.3.0

```
npm ERR! v8-debug@0.4.6 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the v8-debug@0.4.6 install script 'node-pre-gyp install --fallback-to-build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the v8-debug package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get their info via:
npm ERR!     npm owner ls v8-debug
npm ERR! There is likely additional logging output above.
```

Seems that the version of grunt-node-inspector used in this case (^0.1.5) uses an outdated version of node-inspector. More info here: https://github.com/node-inspector/v8-debug/issues/7

Updating grunt-node-inspector to version 0.4.1 seems to fix the problem at least on my setup.

--- a/app/templates/_package.json
+++ b/app/templates/_package.json
@@ -25,7 +25,7 @@
     "jshint-stylish": "^2.0.1",
     "load-grunt-tasks": "^0.4.0",
     "load-grunt-configs": "^0.4.1",
-    "grunt-node-inspector": "^0.1.5",
+    "grunt-node-inspector": "^0.4.1",
     "time-grunt": "^0.3.1",
     "grunt-concurrent": "^0.5.0",
     "grunt-nodemon": "^0.2.1", <% if (preprocessor === 'sass') { %>

